pyproject.toml

```diff
@@ -25,7 +25,8 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-    "pooch"
+    "pooch",
+    "shapely"
```

Comment:
We need to keep this lean. I know that in practice, people will likely have shapely installed anyway but we don't strictly need it.